doc/Parameters.md

```diff
@@ -0,0 +1,115 @@
+# Options for tinyRNA Tools
+This page provides an explanation of the parameters offered by each of our Python components. We represent each parameter with its Run Config key and its corresponding commandline argument. When running the tinyRNA pipeline (`tiny run/recount/replot`), you'll use the Run Config key to specify your preferences. The commandline argument is used when you run a tool as an individual, standalone step (`tiny-collapse`, `tiny-count`, or `tiny-plot`)
+
```

Comment:
Can we give the simplest example of how to run each tool as a standalone step?  Or maybe I haven't got to it yet.

Reply:
The corresponding tool's documentation will have an explanation of what is required for a standalone run. Right now Counter.md has this. Plotter.md does as well but it's included in the Input Files section